### PR TITLE
Limit Markov message length

### PR DESCRIPTION
**Pull request by ForNeVeR**
I've changed Platonus API and added its usage to the Horta code.

There's no need in additional tests because it's Platonus responsibility to test this API (and it does that indeed).

Also I've updated our scalac and sbt settings, and also expanded a bit our EditorConfig. I don't see a point of creating distinct PR with just these changes, so in a lazy manner I've included them here.

### CODE REVIEW

**Comment by Newlifer** [comment]
I second this!

**Comment by hagane** [comment]
i third this!

File: build.sbt [hunk 1]
```diff
@@ -14,7 +14,7 @@ libraryDependencies ++= Seq(
   "org.scalikejdbc" %% "scalikejdbc" % "2.1.2",
   "org.scalatest" %% "scalatest" % "2.1.5" % "test",
   "com.googlecode.flyway" % "flyway-core" % "2.3",
-  "me.fornever" %% "platonus" % "0.2.1",
+  "me.fornever" %% "platonus" % "0.3",
```
Comment (rexim) [inline build.sbt]: But who reviewed platanus 0.3?

Comment (rexim) [inline build.sbt]: Can we trust him?

**Comment by rexim** [comment]
The shit looks good to me. Approved!

File: build.sbt (hunk 1 above)
Comment (ForNeVeR) [inline build.sbt]: I think its author have reviewed it. Do you mean you want it to be moved to codingteam foundation?

Comment (ForNeVeR) [inline build.sbt]: You know, we have the _rules_ these days. If'd there be two or more codingteam contributors (currently only me) - I would be ready to transfer it.

**Comment by ForNeVeR** [comment]
Alrighty then, thank you!

**Pull request merged**